internal/v1/server_test.go

```diff
@@ -207,7 +208,9 @@ func startServer(t *testing.T, tscc *testServerClientsConf, conf *ServerConfig)
 	URL := "http://" + addr
 	go func() {
 		err = echoServer.Start(addr)
-		require.Equal(t, err, http.ErrServerClosed)
+		if !errors.Is(err, http.ErrServerClosed) {
+			panic(fmt.Errorf("starting test server failed %w", err))
```

Comment:
can none of the testing functions be of help here? Like https://pkg.go.dev/testing#B.Fatalf or something? might be nicer than a panic.

Reply:
In my tests with `t.Fatalf()` the code still tries to connect to the wrong server (the one who has the port bound)
So does not stop the tests fast enough